packages/transaction_signing_gateway/lib/transaction_summary_ui.dart

```diff
@@ -3,6 +3,12 @@ import 'package:transaction_signing_gateway/model/transaction_signing_failure.da
 import 'package:transaction_signing_gateway/model/unsigned_transaction.dart';
 
 abstract class TransactionSummaryUI {
+  /// Shows a summary of the transaction for the user to approve.
+  ///
+  /// Implementations of this method should display a summary of the transaction that is about to be signed.
+  /// It should allow the user to review all the relevant details and present him with options to either approve
+  /// or decline the transaction. If user declines the transaction returned future should exit with
```

Comment:
```suggestion
  /// This method allows the user to review all of the relevant details and present them with options to approve
    /// or decline the transaction. If the transaction is declined, the transaction exits with
```

Reply:
singular "they" is acceptable in 2021 (fun, right!?). 
I wasn't sure what we are doing with exit, do we mean "the transaction exits with" or?

Reply:
here I meant that the returned [Future] object emits either a `left(...)` in case of an error or `right(..)` when the operation is successful. I've rephrased the sentence to this:
```suggestion
  /// This method allows the user to review all of the relevant details and present them with options to approve
  /// or decline the transaction. If the transaction is declined, the returned [Future] emits
```